sdk/appcenter/src/main/java/com/microsoft/appcenter/Constants.java

```diff
@@ -7,6 +7,7 @@
 
 import android.content.Context;
 import android.content.pm.ApplicationInfo;
+import android.support.annotation.VisibleForTesting;
```

Comment:
Unused import?

Please consider using the Android Studio commit dialog that has all the actions to check code analysis, clean imports, format code etc....

Reply:
This comment was not resolved... Will fix it.